workers/shared.py

```diff
@@ -92,6 +92,8 @@ def guess_ensembl_release(filepath):
         release = varcode.load_vcf(filepath)[0].ensembl.release
     except ValueError:  # no guesses from varcode, return default
         release = config.ENSEMBL_RELEASE
+    except Exception, error:  # varcode cannot handle this one, so go w/ default
```

Comment:
Probably don't need to capture the exception object (i.e. you can drop the `, error`) unless there's a reason to catch it?